sections/03_categorize.md

```diff
@@ -46,65 +46,167 @@ regards to data availability and labeling.
+even when dataset size is not big enough to train large capacity models from
+scratch [@tag:Bar2015_nonmed_tl @tag:Shin2016_cad_tl
+@tag:Rajkomar2017_radiographs @tag:Lakhani2017_radiography]. All these
+studies demonstrate successful transfer of features learnt from natural image
+datasets, such as ImageNet [@tag:Russakovsky2015_imagenet]. Rajkomar et al.
+[@tag:Rajkomar2017_radiographs] showed that although a deep CNN trained on
+natural images can boost radiology image classification performance, target
+task required either re-training the initial model from scratch with special
+pre-processing or fine-tuning of the whole network on radiographs with heavy
+data augmentation to avoid overfitting. Shin et al. [@tag:Shin2016_cad_tl]
+compared various deep network architectures, dataset characteristics, and
+training procedures for computer tomography-based (CT) abnormality detection.
+They concluded that in case of three-dimensional data networks as deep as 22
+layers can be useful for such problems despite the limited size of training
+datasets. However, they note, that choice of a specific architecture,
+parameter setting, and model fine-tuning needed is very problem and
+dataset-specific. Moreover, this type of tasks often depends on both lesion
+localization and appearance that pose challenges for CNN-based approaches.
+Straightforward attempts to capture useful information from full-size images
+in all three dimensions simultaneously make applications of standard neural
+network architectures computationally unfeasible due to the curse of
+dimensionality. Instead, two dimensional models are often used to either process
+image slices individually (2D), or aggregate information from a number of 2D
+projections in the native space (2.5D). Roth et al. compared 2D, 2.5D, and
+3D CNNs on a number of tasks for computer-aided detection from CT scans and
+showed that 2.5D CNNs performed comparably well to 3D analogs, while requiring
+much less training time, especially on augmented training sets
+[@tag:Roth2015_view_agg_cad]. Another advantage of 2D and 2.5D networks is a
+possibility to use widely available models pre-trained on natural images.
+
+Similarly, in magnetic resonance image (MRI) analysis, limited size of training
+sets and large dimensionality represent challenges to deep learning
+applications. For example, Amit et al. [@tag:Amit2017_breast_mri] investigated
+the tradeoffs between using pre-trained models from a different domain and
+retraining a small-size CNN on MRI images. They showed that smaller network
+trained with sufficient data augmentation on few hundred images from a
+few dozen patients can outperform a pre-trained out-of-domain classifier.
+Nie et al. [@tag:Nie2016_3d_survival] showed that multimodal, multi-channel
+3D deep architecture was successful at learning high-level brain tumor
+appearance features jointly from T1 MRI, fMRI and DTI images, outperforming
```

Comment:
Do we have these abbreviations defined and do we need them. Maybe just says "MRI"?